portfolio/src/main/webapp/script.js

```diff
@@ -77,19 +77,20 @@ const fetchMessageUsingJSON = () => {
     // of messagesObj
     const messagesSize = messagesObj.messages.length;
     const message = messagesObj.messages[Math.floor(Math.random() * messagesSize)];
-    
-    // Add message to the page.
+
+    // Add message to the page
     factContainer.innerText = message;
   });
 };
 
 const showUserComments = commentLimit => {
-  fetch('/data?num-comments=' + commentLimit).then(response => response.json()).then(comments => {
+  fetch('/data?num-comments=' + commentLimit).then(response => response.json()).then(commentsObj => {
 
     // Build the comments setion with all of the user comments, one after the other
     commentsSection.innerHTML = "";
```

Comment:
either use backticks or single quotes everywhere in js, unless you need to escape more quotes.

Reply:
Oh do you mean on line 90? I gotcha. Changed!

Reply:
oh sorry-- meant:

```
    commentsSection.innerHTML = "";
```

should be

```
    commentsSection.innerHTML = '';
```
 or something

Reply:
Gotcha! Already changed. I'm currently seeing if the Map with Gson works the way I want it to